src/Batch/Batch.Test/BatchTestHelpers.cs

```diff
@@ -81,7 +81,7 @@ public static class BatchTestHelpers
                 tags: tags == null ? null : TagsConversionHelper.CreateTagDictionary(tags, true),
                 dedicatedCoreQuotaPerVMFamilyEnforced: dedicatedCoreQuotaPerVMFamilyEnforced,
                 dedicatedCoreQuotaPerVMFamily: machineFamilyQuotas,
-                identity: identity ?? new BatchAccountIdentity(ResourceIdentityType.None));
+                identity: identity ?? new BatchAccountIdentity(ResourceIdentityType.None, null, null, null as IDictionary<string, UserAssignedIdentities>));
```

Comment:
Can we just set null here instead of an odd empty BatchAccountIdentity object? I don't think we want customers to construct an empty object like that, so IMO it'd be better to have the tests use a more common case.

Reply:
I mean in the case where an identity is not being used of course.

Reply:
Squashed my changes. Here's the file: https://github.com/Azure/azure-powershell/blob/eafed1824eb5c4ec1e2c1b57fdd823eab74b434e/src/Batch/Batch.Test/BatchTestHelpers.cs